packages/ui-predicate-vue/src/ui-predicate.vue

```diff
@@ -61,6 +65,12 @@ export default {
       setArgumentValue: function(predicate, value) {
         return vm.ctrl.setArgumentValue(predicate, value);
       },
+      getUIComponent(name) {
+        // Could not be ready
```

Comment:
on oubliera pourquoi ce commentaire est là dans 1 mois, est-ce qu'on pourrait plutôt expliciter le problème et pourquoi il y a cette condition par exemple :

```
// getUIComponent will be called before mount time (because VueJS will initialize every components and call getUIComponent() for some and `ctrl` (instance of UIPredicateCore) will only be available at a later time *after* mount() time. Thus, it was throwing an error at boot time because vm.ctrl was undefined. 
// here we yield undefined if vm.ctrl does not exist, thus rendering nothing. Some milliseconds later, once mounted() has been called and vm.ctrl has been set asynchronously, components will be automatically re-rendered with the predicates data and the user won't have seen the empty spaces :)
```